docs/CMakeLists.txt

```diff
@@ -379,208 +381,188 @@ function(ivw_private_make_help)
         COMMENT "Building Qt QCH files for ${HARG_NAME}"
         VERBATIM
     )
-    set_target_properties("DOXY-HELP-${HARG_NAME}" PROPERTIES FOLDER "doc/qch" EXCLUDE_FROM_ALL TRUE)
+    set_target_properties("DOXY-HELP-${HARG_NAME}" PROPERTIES FOLDER "docs/qch" EXCLUDE_FROM_ALL TRUE)
 endfunction()
 
-function(make_doxygen_target modules_var)
-    if(NOT ${IVW_DOXYGEN_PROJECT})
-        return()
-    endif()
-
-    find_package(Perl QUIET)            # sets: PERL_FOUND, PERL_EXECUTABLE
-    find_package(Doxygen QUIET)         # sets: DOXYGEN_FOUND, DOXYGEN_EXECUTABLE,
-                                        # DOXYGEN_DOT_FOUND, DOXYGEN_DOT_EXECUTABLE
-    find_package(PythonInterp QUIET)    # sets: PYTHONINTERP_FOUND PYTHON_EXECUTABLE
+if(NOT ${IVW_DOXYGEN_PROJECT})
+    return()
+endif()
 
-    if(NOT ${DOXYGEN_FOUND})
-    message(WARNING "Tried to create doxygen project, but doxygen was not found")
-        return()
-    endif()
+ivw_retrieve_all_modules(reg_modules)
+ivw_mod_name_to_mod_dep(modules ${reg_modules})
 
-    if(${DOXYGEN_DOT_FOUND})
-        get_filename_component(DOXYGEN_DOT_PATH ${DOXYGEN_DOT_EXECUTABLE} PATH)
+find_package(Perl QUIET)            # sets: PERL_FOUND, PERL_EXECUTABLE
+find_package(Doxygen QUIET)         # sets: DOXYGEN_FOUND, DOXYGEN_EXECUTABLE,
+                                    # DOXYGEN_DOT_FOUND, DOXYGEN_DOT_EXECUTABLE
+find_package(PythonInterp QUIET)    # sets: PYTHONINTERP_FOUND PYTHON_EXECUTABLE
+if(NOT ${DOXYGEN_FOUND})
+message(WARNING "Tried to create doxygen project, but doxygen was not found")
+    return()
+endif()
+if(${DOXYGEN_DOT_FOUND})
+    get_filename_component(DOXYGEN_DOT_PATH ${DOXYGEN_DOT_EXECUTABLE} PATH)
+endif()
+set(ivw_doxy_dir ${IVW_ROOT_DIR}/docs)
+set(output_dir ${CMAKE_CURRENT_BINARY_DIR})
+set(tag_files "")
+set(dependency_list "")
+set(GENERATE_IMG "YES")
+if(PYTHONINTERP_FOUND)
+    if(${MSVC})
+        set(PREFIX_PYTHON "python ")
+    else()
+        set(PREFIX_PYTHON "python3 ")
     endif()
-
-    set(ivw_doxy_dir ${IVW_ROOT_DIR}/docs)
-    set(IVW_DOXYGEN_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/doc"
-        CACHE PATH "Path to put the doxygen output")
-    set(output_dir ${IVW_DOXYGEN_OUTPUT_DIRECTORY})
-    set(tag_files "")
-    set(dependency_list "")
-
-    set(GENERATE_IMG "YES")
-
-    if(PYTHONINTERP_FOUND)
-        if(${MSVC})
-            set(PREFIX_PYTHON "python ")
-        else()
-            set(PREFIX_PYTHON "python3 ")
+    #string(REPLACE " " "\ " SPACE_PYTHON ${PYTHON_EXECUTABLE})
```

Comment:
Remove

Reply:
This is some old stuff, that I did not comment out, and it seems a bit shady, so I left it until I have time to investigate it...